dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/io/readers/SubSnpCoreFieldsReaderTest.java

```diff
@@ -135,12 +127,39 @@ public void setUp() {
                                                  1766472L,
                                                  1766472L,
                                                  1,
+                                                 LocationType.SNP,
                                                  "4",
                                                  91223961L,
                                                  91223961L,
                                                  "T", "T", "C", "T/A",
                                                  "NC_006091.4:g.91223961T>C", 91223961L, 91223961L, 1,
                                                  "NT_455866.1:g.1766472T>C", 1766472L, 1766472L, 1));
+
+//        expectedSubsnps.add(new SubSnpCoreFields(28450714,
+//                                                 17870373L,
+//                                                 LocationType.INSERTION,
+//                                                 1,
+//                                                 "NW_014646631.1",
+//                                                 3283133L,
+//                                                 3283134L,
+//                                                 1,
+//                                                 "24",
+//                                                 3776507L,
+//                                                 3776508L
+//        ));
+//
+//        expectedSubsnps.add(new SubSnpCoreFields(28450575,
+//                                                 17870234L,
+//                                                 LocationType.DELETION,
+//                                                 1,
+//                                                 "NW_014646451.1",
+//                                                 3468524L,
+//                                                 3468527L,
+//                                                 1,
+//                                                 "19",
+//                                                 23526721L,
+//                                                 23526724L
+//        ));
```

Comment:
If these are needed the tests need to be adapted accordingly, otherwise please remove commented block.